recipe/meta.yaml

```diff
@@ -22,9 +22,9 @@ requirements:
   build:
     - cmake
     - make  # [not win]
-    - {{ stdlib('c') }}
```

Comment:
Is `- {{ stdlib('c') }}` we don't use anymore?

Reply:
We don't have support for it yet.